src/main/clojure/clara/rules/compiler.clj

```diff
@@ -73,6 +73,12 @@
                         ;; Function that takes facts and determines what alpha nodes they match.
                         get-alphas-fn])
 
+(defn- is-variable?
```

Comment:
Dont we call these "unification variables/bindings" elsewhere? variable seems...vaguer than its actual intent.